### ISSUE TITLE
Cannot view old stats with !gstats if game mode player limits are changed

### ISSUE THREAD

**Issue by nyuszika7h** [lykoss/lykos#267]
The alpha game mode was recently changed to require 10p, which makes it impossible to view 7 to 9p alpha game stats with `!gstats`:

```
17:57:04 -- lykos: Please enter an integer between 10 and 24.
```

**Comment by jacob1** [comment]
This happens with removed roles and game modes too. For player coutns it is a little simpler, we can probably just allow invalid numbers, and it would return no results, which should be fine (perhaps it should only say the range is invalid after getting no results back).

For roles / game modes we do autocorrect. Maybe on startup we could get a list of valid things to autocorrect to for those. Or just always allow anything to go through (while still attempting autocorrection - I don't think we have any fake roles / game modes which are a partial match of real ones)

**Comment by skizzerz** [comment]
Fixed in 4cf81714262b4190104b8db669c7504b48f166f3 (removed roles and removed game modes will now show up too provided there's some local bot config to re-enable stats for them)
